Make package cross platform

What is the best way to make the package work on Windows and the other supported CRAN platforms? Ideally we would not need to bundle the cbc code with the package.

@jeroen there is one warning actually if I run `devtools::build_win(version = "R-devel")`:

>  checking whether package 'rcbc' can be installed ... WARNING
> Found the following significant warnings:
>   cpp_cbc_solve.cpp:47:29: warning: ISO C++ forbids variable length array 'argList' [-Wvla]
> See 'd:/RCompile/CRANguest/R-devel/rcbc.Rcheck/00install.out' for details.

I am not an expert on c/c++, is that easy to fix?

--- a/DESCRIPTION
+++ b/DESCRIPTION
@@ -10,6 +10,7 @@ Authors@R: c(
 License: GPL-3 + file LICENSE
 NeedsCompilation: yes
 SystemRequirements: CBC libraries and headers
+Biarch: true
 Encoding: UTF-8
 LazyData: true
 LinkingTo: Rcpp

--- a/configure.win
+++ b/configure.win
@@ -1,0 +1,2 @@
+# No configuration needed on Windows
+# (this suppresses an unnecessary warning message during installation)

--- a/src/Makevars.win
+++ b/src/Makevars.win
@@ -1,5 +1,9 @@
+VERSION = 4.9.3
+RWINLIB = ../windows/cbc-2.9.8/lib-$(VERSION)$(R_ARCH)
+
 PKG_CPPFLAGS = -I../windows/cbc-2.9.8/include/coin
-PKG_LIBS = -L../windows/cbc-2.9.8/lib${subst gcc,,${COMPILED_BY}}${R_ARCH} \
+
+PKG_LIBS = -L$(RWINLIB) \
 	-lCbcSolver -lClpSolver \
 	-lOsiCbc -lCbc -lCgl -lOsiClp -lClp -lOsi -lCoinUtils -lz -lm
 

